src/components/molecules/InboxItem.vue

```diff
@@ -15,35 +15,55 @@
         v-if="inboxItem.senderIcon"
         class="w-12 h-12 md:w-12 md:h-12"
         :src="icons[inboxItem.senderIcon]"
-        alt="Inbox icon"
+        alt="Inbox icon."
```

Comment:
Is it possible to have an empty quote here for the alt text as this icon doesn't do anything, it is not clickable and inbox icon isn't relevant for the assistant and it isn't translated.

Reply:
I'll leave it as is for now, if Neil says that we should/can then we can change it in a later PR. I agree with you, the sender name is enough and (in my opinion) if we do decide to mention the icons, then we should be giving a full description of the icon regardless, which isn't very practical in this case.